ui/appui-react/src/appui-react/ui-items-provider/UiItemsManager.ts

```diff
@@ -220,7 +224,7 @@ export class UiItemsManager {
    * @returns An array of widgets.
    */
   public static getWidgets(stageId: string, stageUsage: string, location: StagePanelLocation, section?: StagePanelSection): ReadonlyArray<ProviderItem<Widget>> {
-    const widgets: ProviderItem<Widget>[] = [];
+    const widgets: ProviderItem<Widget>[] = this._abstractHook.getWidgets(stageId, stageUsage, location, section);
```

Comment:
Should we just fallback to using **abstract** `UiItemsManager` if one is defined in `appui-abstract`?
I.e. would following work with current impl?
1. `appui-abstract` 3.7 registers a widget **B**.
2. `appui-react` 4.0 registers a widget **B**.
3. Calling `getWidget` in both `appui-react` and `appui-abstract` should return [**A**, **B**].

Reply:
We can do that as well, but we'll need to handle conversion both ways (When using React, we'll need to convert from and to Abstract, and if it does not exist, not conversion is needed)